tests/__init__.py

```diff
@@ -3,3 +3,20 @@
 #
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
+
+# Check at the top-level that torchao is installed.
+# This is better than doing it at every import site.
+# We have to do this because it is not currently possible to
+# properly support both nightly and stable installs of PyTorch + torchao
+# in pyproject.toml.
+try:
+    import torchao  # noqa
+except ImportError as e:
+    raise ImportError(
+        """
+    torchao not installed.
+    If you are using stable PyTorch please install via pip install torchao.
+    If you are using nightly PyTorch please install via
+    pip install --pre torchao --index-url https://download.pytorch.org/whl/nightly/cu121."
```

Comment:
Maybe just point to the install page so we only have to maintain one source of truth for how to install torchao.